packages/store/reducers.js

```diff
@@ -5,7 +5,8 @@ import { reducer as tabsReducer } from '@bufferapp/publish-tabs';
 import { reducer as queueReducer } from '@bufferapp/publish-queue';
 import { reducer as sentReducer } from '@bufferapp/publish-sent';
 import { reducer as draftsReducer } from '@bufferapp/publish-drafts';
-import { reducer as settingsReducer } from '@bufferapp/publish-settings';
+import { reducer as PostingScheduleReducer } from '@bufferapp/publish-posting-schedule';
```

Comment:
Hey @lolaodelola, I think that these should probably be lowercase for consistency here 😄 

```suggestion
import { reducer as postingScheduleReducer } from '@bufferapp/publish-posting-schedule';
```